server/controller.ts

```diff
@@ -35,11 +39,13 @@ const validateNote = (note) => {
 };
 
 export const saveNote = async (req: Request, res: Response): Promise<void> => {
+  logger.debug(req, "controller, saveNote, req");
```

Comment:
Requests are tend to be very large, better log only the required parts like body or params